src/main/java/com/example/moviemanager/controller/dto/CreateMovieRequest.java

```diff
@@ -0,0 +1,27 @@
+package com.example.moviemanager.controller.dto;
+
+import com.example.moviemanager.repository.model.Movie;
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Size;
+
+public record CreateMovieRequest
+        (@NotBlank(message = "Title is mandatory.")
```

Comment:
Opening bracket should be on previous line and closing bracket (line 22) should be on new line